clement-moulin-frier/vivarium
# Added max_speed attribute to agents

## Description
As the title says

## Related Issue (if applicable)
closes #65

## How to test
- Launch server and interface
- select an agent, set it on `MANUAL`, set both its motors to 1 and put its wheel diameter to 10 (to make it go fast)
- then lower the `max_speed` setting, it should make the agent slow down
